### ISSUE TITLE
Prevent search engines from indexing profile pages in ALPHA release

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#1101]
To ensure that we do not negatively impact our recent SEO success, we should restrict all search engines from indexing the `alpha` branch.

While we could do this with a robots.txt file with `User-agent: * Disallow: /`, it is preferable to block the page using the `<meta name='robots'>` tag. 

>If the page is blocked by a robots.txt file, the crawler will never see the noindex directive, and the page can still appear in search results, for example if other pages link to it.
Source: [Google Search Console documentation](https://support.google.com/webmasters/answer/93710?hl=en)

Because we will have the production Angular app linking to the `alpha` branch, the robots.txt approach could result in crawlers indexing the page as they will be able to follow the links on the Angular version profile pages.

Therefore, can we please add the `<meta name="robots" content="noindex" />` tag to the template used to generate each page?

**Comment by andrewhercules** [comment]
Ticket closed - merged into `alpha`
